mixins/arrow-keys-mixin.md

```diff
@@ -12,14 +12,14 @@ If the elements are known up-front and are in the element's local DOM scope, sim
 import { ArrowKeysMixin } from '@brightspace-ui/core/mixins/arrow-keys-mixin.js';
 class MyElement extends ArrowKeysMixin(LitElement) {
   render() {
-    return html`
+    return this.arrowKeysContainer(html`
```

Comment:
Yeah, I agree this needs to be added.

The alternative is that the consumer wires up the handler themselves, [which is done in at least one place](https://github.com/Brightspace/folio-app/blob/main/src/web-components/radio-button/radio-button-group.js#L72).  In this case, the `host` is the container, but generally, I'd rather see them use `arrowKeysContainer()` so we can better contain the implementation details.